README.md

```diff
@@ -78,19 +78,34 @@ yourlogger.addHandler(airbrake.AirbrakeHandler())
 _by default, the `AirbrakeHandler` only handles logs level ERROR (40) and above_
 
 #### Additional Options
-More options are available to configure this library. Similar to setup, you can do this with environment variables
+More options are available to configure this library. 
+
+For example, you can send the hostname to add more context to your errors.  
+One way is by setting the HOSTNAME env var.
 ```
 export HOSTNAME=sassbox-101.prod.api
 ```
-Or when you instantiate the logger
+Or you can set it more explicitly when you instantiate the logger.
 ```python
 import airbrake
 
 logger = airbrake.getLogger(api_key=*****, project_id=12345, hostname='sassbox-101.prod.api')
 ```
 
 The available options are:
-- hostname (AIRBRAKE_HOSTNAME), default: `socket.gethostname()`
+- hostname, defaults to env var `HOSTNAME` or `socket.gethostname()`
+- environment, defaults to env var `AIRBRAKE_ENVIRONMENT`
+- base_url, defaults to env var `AIRBRAKE_BASE_URL` or https://airbrake.io
+- component, defaults to None
+- action, defaults to None
+- app_version, defaults to None
+- app_url, defaults to None
+- user_agent, defaults to None
+- root_directory, defaults to None
+- user_id, defaults to None
+- user_name, defaults to None
+- user_email, defaults to None
```

Comment:
Since this is more a core library, I don't think we have anything to go on in terms of a "user" concept to use in automatic way. If other integrations build on this, and within those frameworks they have a user concept, than we can leverage it there. I do think this makes sense to have here.